### PR TITLE
[refactor] Cleanup Python requirements

### PR DESCRIPTION
**Pull request by devfox-se**

### CODE REVIEW

**Comment by devfox-se** [comment]
Covers #1650

File: setup.py [hunk 1]
```diff
@@ -51,25 +51,23 @@ def package_files(directory):
...
     'filelock>=3.0.0',
-    'GitPython>=3.0.4',
     'numpy>=1.12.0',
-    'packaging>=19.0',
     'protobuf>=3.11.0',
     'psutil>=5.6.7',
     'py3nvml>=0.2.5',
     'RestrictedPython>=5.1',
     'tqdm>=4.20.0',
-    'aiofiles>=0.5.0',
```
Comment (mihran113) [inline setup.py:62]: aiofiles is required to use `FileResponse` to serve static files

**Review by mihran113 — CHANGES_REQUESTED** [review]
Left a comment there, other than that looks great 👍

**Review by alberttorosyan — APPROVED** [review]
lgtm

**Comment by alberttorosyan** [comment]
Please rename to
`[refactor] Cleanup Python requirements` so it matches the commit message format

**Review by mihran113 — APPROVED** [review]

**Pull request merged**